### PR TITLE
chore: customization of fee show

### PR DESCRIPTION
**Pull request by Michellah**

### CODE REVIEW

**Comment by Michellah** [comment]
fee

**Comment by Michellah** [comment]
image

File: package.json
Comment (JeanMarc-RAJAONARIVELONA) [inline package.json:1]: remove package you don't use it

Comment (JeanMarc-RAJAONARIVELONA) [reply package.json:1]: remov

File: src/operations/fees/utils/gridStyle.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {PALETTE_COLORS} from "@/haTheme";
+
+export const gridStyle = () => ({
```
Comment (JeanMarc-RAJAONARIVELONA) [inline src/operations/fees/utils/gridStyle.jsx]: UPPER_SNAKE_CASE for every non-temporary constant use it for all project

**Pull request merged**